tests/lora/conftest.py

```diff
@@ -152,6 +152,18 @@ def sql_lora_files(sql_lora_huggingface_id):
     return snapshot_download(repo_id=sql_lora_huggingface_id)
 
 
+@pytest.fixture(scope="session")
+def llama3_1_8b_chess_lora():
+    return snapshot_download(
+        repo_id="mkopecki/chess-lora-adapter-llama-3.1-8b")
+
+
+@pytest.fixture(scope="session")
+def llama3_1_8b_ultravox_chess_lora():
+    # ultravox chess lora is result of transformation of above chess llama lora
+    return snapshot_download(repo_id="thedebugger11/ultravox-chess-lora")
```

Comment:
generated using this script: https://github.com/thedebugger/ultravox-lora-scipts/blob/main/peft_ultravox_transform.py

Reply:
This is done to align module names in the safetensor adapter file `base_model.model` -> `base_model.model.language_model`

Reply:
So if anyone wants to use llama lora with ultravox, it can do that with this after doing this transformation